smoke-test.sh

```diff
@@ -198,6 +205,17 @@ function doCommand() {
             $killCommand
             wait $!
             echo "    terminated with exit code $?" | tee -a "$logFile"
+        elif [ "$1" == "publish" ]; then
+            binlogPrefix=$(echo "$testingDir/${lang}_${proj}_${binlogOnlinePart}_${binlogHttpsPart}_" | sed 's/--/_/g ; s/ /_/g')
```

Comment:
Instead of duplicating, change the earlier declaration to support this?

```sh
binlogPrefix="$testingDir/${lang}_${proj}_${binlogOnlinePart}_${binlogHttpsPart}_"
binlog="${binlogPrefix}${1}binlog.binlog"
```

Reply:
Thanks, changed as suggested.